lib/rubocop/cop/lint/shadowing_outer_local_variable.rb

```diff
@@ -8,6 +8,9 @@ module Lint
       # given by `ruby -cw` prior to Ruby 2.6:
       # "shadowing outer local variable - foo".
       #
+      # Note: Shadowing of variables in block passed to `Ractor.new` is allowed
```

Comment:
`Note:` -> `NOTE:`, so AsciiDoc would interpret it properly.

Reply:
I'm guessing you can also add some example with `Ractor.new` and extend the note saying this applies only to Ruby 3+.

Reply:
Added